### PR TITLE
Manifest schema updates

### PR DESCRIPTION
**Pull request by dwapstra**

### CODE REVIEW

**Review by simingy — APPROVED** [review]

File: docs/manifest/schema.rst [hunk 1]
```diff
@@ -81,3 +83,22 @@ Manifest Schema
...
                          # (optional)
                 <key>: <value>
+
+            max_runtime: HH:MM,   # max runtime in hours:minutes (optional)
+
+            interest_list:       # list of email addresses (optional)
+                - list@domain
+                - user@domain
+
+            priority: #  job priority (integer) (optional)
```
Comment (jeaubin) [inline docs/manifest/schema.rst:93]: lower = top or highest = top ?

And how is it used?

File: docs/manifest/schema.rst [hunk 2]
```diff
@@ -81,3 +83,22 @@ Manifest Schema
...
+
+            testbed:   # (optional)
+                name:  # name of the testbed
+
+            topology:  # (optional)
```
Comment (jeaubin) [inline docs/manifest/schema.rst:98]: difference between topology vs testbed

File: docs/manifest/schema.rst [hunk 3]
```diff
@@ -81,3 +83,22 @@ Manifest Schema
...
+                name:  # name of the topology
+
+            docker:  # docker details  (optional)
+                volumes:  # docker volume mounts
```
Comment (jeaubin) [inline docs/manifest/schema.rst:102]: i feel this could lead to more questions 

Is there more documentation on this?

**Review by jeaubin — APPROVED** [review]

File: docs/manifest/schema.rst (hunk 3 above)
Comment (dwapstra) [reply docs/manifest/schema.rst:106]: No docs on xpresso other than mention of "Cloud Mount Points" under job execution

File: docs/manifest/schema.rst (hunk 2 above)
Comment (dwapstra) [reply docs/manifest/schema.rst]: updated comment

File: docs/manifest/schema.rst (hunk 1 above)
Comment (dwapstra) [reply docs/manifest/schema.rst]: added prio description

**Pull request merged**